aten/src/ATen/core/boxing/impl/make_boxed_from_unboxed_functor.h

```diff
@@ -328,13 +329,51 @@ namespace impl {
     ))...);
   }
 
+  template<class Functor, bool AllowDeprecatedTypes, size_t... ivalue_arg_indices>
```

Comment:
You shouldn't need to special case `call_functor_with_args_from_stack_` because you modified `wrap_kernel_functor_unboxed` to drop the argument and `call_functor_with_args_from_stack_` only calls the things already wrapped by `wrap_kernel_functor_unboxed`.